README.md

```diff
@@ -56,11 +56,14 @@
 - [Killed by Google](https://github.com/codyogden/killedbygoogle)
 - [Open source drinks](https://github.com/alfg/opendrinks)
 - [Predator](https://github.com/Zooz/predator)
+-[Python Mini-Project]
```

Comment:
Can you add an empty space between the "-" and the "[" ?

Also remove the breakline, they should be in the same line